lib/osf-components/addon/components/addons-service/configured-addon-edit/component.ts

```diff
@@ -23,12 +24,16 @@ export default class ConfiguredAddonEdit extends Component<Args> {
         itemType: ItemType.Folder,
     };
 
+    get hasRootFolder() {
+        return !(this.args.authorizedAccount instanceof AuthorizedComputingAccountModel);
+    }
+
     get invalidDisplayName() {
         return !this.displayName || this.displayName?.trim().length === 0;
     }
 
     get disableSave() {
-        return !this.selectedFolder || this.invalidDisplayName || this.args.onSave.isRunning;
+        return this.hasRootFolder && (!this.selectedFolder || this.invalidDisplayName || this.args.onSave.isRunning);
```

Comment:
Would this condition always return `false` when `this.args.authorizedAccount` is an AuthorizedComputingAccount, meaning the save button is always enabled? Or have I confused myself with the logic?

I _think_ it should be something like this (I've rearranged the checks to make it a little easier for myself):
```suggestion
        return this.invalidDisplayName || this.args.onSave.isRunning || (this.hasRootFolder && !this.selectedFolder) || ;
```

Reply:
D'oh! Absolutely right. I will fix that up.